File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout;
using DevExpress.XtraEditors;

namespace WindowsApplication1 {
	public partial class Form1 : Form {
		public Form1() {
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e) {
			int rows = 10;
			int cols = 5;
			layoutControl1.BeginInit();
			List<BaseLayoutItem> layout = new List<BaseLayoutItem>();
			for(int i = 0; i < rows; ++i) {
				for(int j = 0; j < cols; ++j) {
					LayoutControlItem labelItems = new LayoutControlItem();
					labelItems.Name = Guid.NewGuid().ToString();
					labelItems.Text = "(" + i + "," + j + ")";
                    TextEdit edit = new TextEdit();
                    edit.Name = Guid.NewGuid().ToString();
                    labelItems.Control = edit;
					layout.Add(labelItems);
					labelItems.Location = new Point(j * 200, i * 200);
					labelItems.Size = new Size(200, 200);
				}
			}
			layoutControlGroup1.Items.AddRange(layout.ToArray());
			layoutControl1.EndInit();
		}
	}
}

File: VB/VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout
Imports DevExpress.XtraEditors

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim rows As Integer = 10
			Dim cols As Integer = 5
			layoutControl1.BeginInit()
			Dim layout As New List(Of BaseLayoutItem)()
			For i As Integer = 0 To rows - 1
				For j As Integer = 0 To cols - 1
					Dim labelItems As New LayoutControlItem()
					labelItems.Name = Guid.NewGuid().ToString()
					labelItems.Text = "(" & i & "," & j & ")"
					Dim edit As New TextEdit()
					edit.Name = Guid.NewGuid().ToString()
					labelItems.Control = edit
					layout.Add(labelItems)
					labelItems.Location = New Point(j * 200, i * 200)
					labelItems.Size = New Size(200, 200)
				Next j
			Next i
			layoutControlGroup1.Items.AddRange(layout.ToArray())
			layoutControl1.EndInit()
		End Sub
	End Class
End Namespace